src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -428,6 +460,12 @@ public bool IsDatabaseLoaded(StringSegment databaseName)
             return false;
         }
 
+        public bool DatabaseRecordExists(string databaseName)
+        {
+            var record = _serverStore.LoadDatabaseRecord(databaseName, out _);
```

Comment:
better to use `ReadRawDatabase` from ServerStore.Cluster

Reply:
done